README.md

```diff
@@ -1,2 +1,68 @@
-# This is for backend service
-A template to create all public facing sites
+# TODO Backend
+
+## Local development setup
+1. Install pyenv
+    - For Mac/Linux - https://github.com/pyenv/pyenv?tab=readme-ov-file#installation
+    - For Windows - https://github.com/pyenv-win/pyenv-win/blob/master/docs/installation.md#chocolatey
+2. Install the configured python version (3.12.7) using pyenv by running the command
+    - For Mac/Linux
+        ```
+        pyenv install
+        ```
+    - For Windows
+        ```
+        pyenv install 3.12.7
+        ```
+3. Create virtual environment by running the command
+    - For Mac/Linux
+        ```
+        pyenv virtualenv 3.12.7 venv
+        ```
+    - For Windows
+        ```
+        python -m pip install virtualenv
+        python -m virtualenv venv
+        ```
+4. Activate the virtual environment by running the command
+    - For Mac/Linux
+        ```
+        pyenv activate venv
+        ```
+    - For Windows
+        ```
+        .\venv\Scripts\activate
+        ```
+5. Install the project dependencies by running the command
+    ```
+    python -m pip install -r requirements.txt
+    ```
+6. Create a `.env` file in the root directory, and copy the content from the `.env.example` file to it
+7. Install [docker](https://docs.docker.com/get-docker/) and [docker compose](https://docs.docker.com/compose/install/)
+8. Start MongoDB using docker
+    ```
+    docker-compose up -d db
```

Comment:
Are you sure about using mongo db? A lot of out of the box solutions would be require fixes.

Reply:
@pallabez 
1. MongoDB has been finalised after discussion with Tejas, Ankush, Yash and Prakash.
2. Can you name some of the solutions that won't work out of the box with MongoDB? One such feature I know about is the Django ORM

Reply:
As mentioned, would recommend setting up a basic crud API to find the nuances. A Todo model with id, name and description should suffice. Check weather admin panel, API contract, migrations works as intended. And how much code it would require.